modules/042-kube-dns/images/coredns/patches/README.md

```diff
@@ -0,0 +1,5 @@
+# Patches
+
+## support-alpha-tolerate-unready-endpoints-annotation.patch
```

Comment:
I think it will be good to open a PR to the coredns itself with an argument, that there are a lot of such charts and services. At least we will see the community reaction and coredns developers' thoughts.

Reply:
It's deprecated from Kubernetes v1.11 and [removed](https://github.com/kubernetes/kubernetes/pull/108020) completely this February. Are you sure about the PR?

Reply:
Yeah, why not.

Reply:
https://github.com/coredns/coredns/pull/5491

Reply:
https://github.com/coredns/coredns/pull/5491